lib/ImportStatements.js

```diff
@@ -9,13 +9,15 @@ import uniqBy from 'lodash.uniqby';
 import Configuration from './Configuration';
```

Comment:
I can buy that. Does ESLint have a commentLineLength? Seems like we wouldn't be the only ones considering a split standard.